src/encoded/static/components/static-pages/components/TableOfContents.js

```diff
@@ -617,44 +622,41 @@ export class TableOfContents extends React.Component {
         );
 
         var marginTop = 0; // Account for test warning
-        if (windowInnerWidth){
```

Comment:
Some refactoring - we get windowWidth and windowHeight passed down from App.js now so no need to access window API here.